Makefile

```diff
@@ -1,16 +1,34 @@
-wasm: 
+POLICY_TYPE ?= opa
+# POLICY_TYPE can be one of `rego`, `opa`, with different build process
+
+SOURCE_FILES := $(shell find . -type f -name '*.rego')
+
+policy.wasm: $(SOURCE_FILES)
+ifndef POLICY_TYPE
+	@printf "Please assign POLICY_TYPE to either rego or opa\n"
+	exit 1
+endif
+ifeq ($(POLICY_TYPE), rego)
+	opa build -t wasm -e policy/violation -o bundle.tar.gz policy.rego
+endif
+ifeq ($(POLICY_TYPE), opa)
 	opa build -t wasm -e policy/main utility/policy.rego -o bundle.tar.gz policy.rego
+endif
```

Comment:
would it be possible to print an error if `POLICY_TYPE` is neither `opa` nor `rego`?

Reply:
Done, force pushed.